tests/test_io.py

```diff
@@ -95,13 +103,16 @@ def test_write_xyz_with_classification(
     np.testing.assert_array_equal(classification_array, read_classification)
 
 
+@mark.dependency(depends=["test_write_xyz_without_classification"])
 def test_read_xyz_without_classification(fixture_path, points_array, colors_array):
     example_filepath = fixture_path / "example_without_classification.xyz"
     points, colors, _ = xyz.read_xyz(example_filepath)
     np.testing.assert_array_equal(points_array, points)
     np.testing.assert_array_equal(colors_array, colors)
+    os.remove(fixture_path / "example_without_classification.xyz")
```

Comment:
```python
(fixture_path / "example_without_classification.xyz").unlink()
```

(Pathlib API instead of `os`, don't forget to remove the `import os` statement in the beginning of the module)